templates/Package_summary.ss

```diff
@@ -0,0 +1,3 @@
+<strong class="package-summary__title">$Title.XML</strong>
+<% if $Description %><br /><% end_if %>
```

Comment:
Can you wrap it in a span and break the line with CSS instead of putting <br> tags in please?